test/command/fixtures/explicit-commands/src/commands.ts

```diff
@@ -0,0 +1,7 @@
+import FooBar from './commands/foo/bar'
+import FooBaz from './commands/foo/baz'
+
+export default {
+  'foo:bar': FooBar,
+  'foo:baz': FooBaz,
```

Comment:
I know you tested my "2 keys pointing at one file" manually, but what about adding it to the fixture so we can't break that behavior once people start depending on it?